docs/en/sql-reference/sql-statements/table_bucket_part_index/CREATE_TABLE.md

```diff
@@ -735,6 +735,29 @@ To disable this feature, you can use the ALTER TABLE statement to set this prope
 ALTER TABLE tbl SET('partition_retention_condition' = '');
 ```
 
+#### Configure flat json config
+
+If you want to use flat json attributes, please specify it in properties. See [ Flat JSON ](../../../using_starrocks/Flat_json.md) for further information
+
```

Comment:
add tips：only support on shared-nothing clusters now

Reply:
Thanks for your review. I have fixed it.